### ISSUE TITLE
Reflow breaks code in Lisp and Lisp-derived languages when reflowing comments

### ISSUE THREAD

**Issue by WalterGR** [atom/autoflow#43]
Lisp and Lisp-derived languages (Common Lisp, Scheme, Clojure, Emacs Lisp, etc.) use semicolons before single-line comments.

Before reflow:

```
;; Yadda yadda yada yada yada yada yada yada yada yada yada yada 
;; blah blah blah.
```

After reflow:

```
;; Yadda yadda yada yada yada yada yada yada yada yada
yada yada ;; blah blah blah.
```

**Comment by 50Wliu** [comment]
Refs #4, #9, #11, #12, #18, #23, #33, #39, ...
I think it's pretty clear that autoflow needs a better way of determining how to wrap things.

**Comment by WalterGR** [comment]
Ah, true.  Apologies for belaboring the point.  :)

**Comment by 50Wliu** [comment]
Sorry, that wasn't meant towards you :).  This is really my first time triaging an autoflow bug, so I was just surprised by all the issues there were.

**Comment by WalterGR** [comment]
Oh, I didn't take it personally.   :)

I'd like to note somewhere that `'` is used for line comments in various Microsoft BASICs including contemporary VB.net and that `REM` is used for line comments in Windows batch files - and neither are supported.  Should I create separate issues for them?  I don't want to add more noise than value...

**Comment by 50Wliu** [comment]
We should probably create a master issue for bugs where autoflow doesn't take into account keeping the code structure intact.

**Comment by 50Wliu** [comment]
Fixed by #45.
